### ISSUE TITLE
Regression in 0.32.0: fatal "no root mount point"

### ISSUE THREAD

**Issue by norbusan** [digint/btrbk#445]
Hi Axel

thanks for the release and the ping on IRC. I tried to set it up but it fails.

Local now works fine as far as I see, but remote root detection now fails, despite having worked before (0.31.N)

The target is set as `ssh://asustor/volume1/_btrbk_/` where `_btrbk_` is just a sub-directory of `/volume1` (not subvolume, not mounted) and `/volume1` is mounted from the remote subvolume `/base`.

I see the following error:
```
___ mountinfo_cache MISS: ssh://asustor
### ssh -i '/etc/btrbk/ssh/id_rsa' root@asustor 'cat /proc/self/mountinfo'
___ [stdout] 1 1 0:2 / / rw - rootfs rootfs rw,size=4014976k,nr_inodes=1003744
___ [stdout] 18 1 0:5 / /proc rw,relatime - proc proc rw
___ [stdout] 19 1 0:17 / /sys rw,relatime - sysfs sysfs rw
___ [stdout] 20 1 0:18 / /dev/pts rw,relatime - devpts devpts rw,gid=5,mode=620,ptmxmode=000
___ [stdout] 21 1 0:19 / /tmp rw,relatime - tmpfs tmpfs rw
___ [stdout] 22 1 9:0 / /volume0 rw,relatime - ext4 /dev/md0 rw
___ [stdout] 23 1 7:0 / /share rw,relatime - ext4 /dev/loop0 rw
___ [stdout] 25 1 0:20 /base /volume1 rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=257,subvol=/base
___ [stdout] 26 25 0:20 /.iscsi /volume1/.@iscsi rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=258,subvol=/.iscsi
___ [stdout] 27 25 0:20 /.@plugins /volume1/.@plugins rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=259,subvol=/.@plugins
___ [stdout] 24 23 0:20 /base/home /share/home rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=257,subvol=/base/home
___ [stdout] 28 23 0:20 /base/Public /share/Public rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=257,subvol=/base/Public
___ [stdout] 29 23 0:20 /base/Web /share/Web rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=257,subvol=/base/Web
___ [stdout] 30 23 0:20 /base/Plex /share/Plex rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=257,subvol=/base/Plex
___ [stdout] 31 23 0:20 /base/PlexMedia /share/PlexMedia rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=257,subvol=/base/PlexMedia
___ [stdout] 32 23 0:20 /base/Backup /share/Backup rw,relatime - btrfs /dev/md1 rw,space_cache=v2,subvolid=257,subvol=/base/Backup
___ [stdout] 35 18 0:28 / /proc/fs/nfsd rw,relatime - nfsd nfsd rw
Command execution successful

ERROR: process died unexpectedly (btrbk v0.32.0)
Please contact the author: Axel Burri <xxxxx>

Stack Trace:
----------------------------------------
no root mount point at ./btrbk line 2972.
 at ./btrbk line 390.
        main::__ANON__("no root mount point at ./btrbk line 2972.\x{a}") called at ./btrbk line 2972
        main::mountinfo_tree(HASH(0x55d3be3c0b20)) called at ./btrbk line 2986
        main::vinfo_mountpoint(HASH(0x55d3be3c0b20), "fs_type", "btrfs") called at ./btrbk line 3025
        main::vinfo_init_root(HASH(0x55d3be3c0b20)) called at ./btrbk line 6630
```

**Comment by digint** [comment]
fixed regression (untested) in 7bff97c438241aaf9716594ea613b410227f6e41

**Comment by digint** [comment]
tested, and merged to master in 0172e4a2406c792404a8d994967a4fd645cb5f44

**Comment by digint** [comment]
included in btrbk-0.32.1
